### ISSUE TITLE
False positive on Common Given steps should be added to Background

### ISSUE THREAD

**Issue by tommywo** [racodond/sonar-gherkin-plugin#71]
Background doesn't support examples, so rule shouldn't apply to steps that require parameters.

**Comment by racodond** [comment]
Hi,

Thanks for your feedback!
Just to make sure I fully understand, could you please provide a code sample?

Thank you

David

**Comment by tommywo** [comment]
Example:
```
Scenario Outline: 
  Given I log in with user <user> and password <password>
  When some action
  Then some result

  Examples:
    | user   | password |
    | user1  | pass1    |
    | user2  | pass2    |

Scenario Outline: 
  Given I log in with user <user> and password <password>
  When different action
  Then different result

  Examples:
    | user   | password |
    | user3  | pass3    |
    | user4  | pass4    |

```
